03_Booleans_Operatörler_Tarih/README.md

```diff
@@ -0,0 +1,444 @@
+
+```js
+let isRaining = true;
+isRaining
+  ? console.log("Yağmurluğa ihtiyacın var.")
+  : console.log("Yağmurluğa ihtiyacın yok.");
+
+isRaining = false;
+isRaining
+  ? console.log("Yağmurluğa ihtiyacın var.")
+  : console.log("Yağmurluğa ihtiyacın yok.");
+```
+
+Sonuç şu şekilde olur:
+
+```sh
+Yağmurluğa ihtiyacın var.
+Yağmurluğa ihtiyacın yok.
+```
+
+```js
+let number = 5;
+number > 0
+  ? console.log(`${number} pozitif bir sayıdır.`)
+  : console.log(`${number} negatif bir sayıdır`);
+number = -5;
+
+number > 0
+  ? console.log(`${number} pozitif bir sayıdır.`)
+  : console.log(`${number} negatif bir sayıdır.`);
+```
+
+Sonuç şu şekilde olur:
+```sh
+5 pozitif bir sayıdır.
+-5 negatif bir sayıdır.
+```
+
+### Operator Precendence (Operatör Öncelikleri)
+
+Operatör önceliklerini bu bağlantıdan okumanızı tavsiye ederim. [link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence)
+
+## Window Methods
+
+### Window alert() method
+
+En başta gördüğünüz gibi alert() metodu, bir mesaj alanı ve bir "Tamam" butonu ile bir uyarı kutusu görüntüler. alert() metodu yerleşik bir metoddur ve değer alır.
+
+```js
+alert(message);
+```
+
+```js
+alert("PROTOTURK'e Hoşgeldin!");
+```
+
+Normal şartlar da alert metodu kullanmamalısınız çünkü kullanışsızdır. Sadece testlerini yaparken kullanmalısınız.
+
+### Window prompt() method
+
+prompt() metodu input kutusu görüntüler ve kullanıcının bu input'a girdiği değeri alır. Bu değeri bir değişkende saklayabilirsiniz. prompt() metodu iki değer alır. İkinci değeri opsiyoneldir.
+
+```js
+prompt("Bir yazı gir", "opsiyonel alan");
+```
+
+```js
+let number = prompt("Rakam gir", "Rakamını buraya gir.");
+console.log(number);
+```
+
+### Window confirm() method
+
+confirm() metodu, mesaj alanı, tamam ve vazgeç butonlarına sahip bir diyalog kutusu görüntüler. confirm kutusu genellikle kullanıcıdan herhangi bir izin almak için kullanılır. confirm() metodu bir string değeri alır. "Tamam" butonuna tıklanırsa true, Vazgeç butonuna tıklanırsa false değer döner. Bu dönen değeri bir değişkende saklayıp kullanabiliriz. 
+
+```js
+const agree = confirm("Silmek istediğinizden emin misiniz? ");
+console.log(agree); // "Tamam" tıklanırsa true, Vazgeç tıklanırsa false gelir.
+```
+
+Tüm metodlar bunlarla sınırlı değil. Ayrı bir bölümde bu metodları derinlemesine işleyeceğiz.
+
+## Date Object (Tarih Nesnesi)
+
+Belirli bir faaliyetin veya olayın zamanını bilmek isteriz. Javascript'te geçerli zaman ve tarih, Date Object kullanılarak oluşturulur.
+
+Oluşturduğumuz Date objesi bize bir çok zaman ve tarih metodu sağlar. Kullandığımız metodlarla tarih ve zaman bilgilerini alabiliriz. Genel de bu metodlar "almak" anlamına gelen _get_ ile başlar.
+_getFullYear(), getMonth(), getDate(), getDay(), getHours(), getMinutes, getSeconds(), getMilliseconds(), getTime(), getDay()_
+
+![Date time Object](../images/date_time_object.png)
```

Comment:
[] => empty
(https://github.com/Asabeneh/30-Days-Of-JavaScript/raw/master/images/date_time_object.png)

Reply:
eyvallah kardo ya çok üşeniyordum bunu yapmaya hahaha